packages/edge-login-ui-rn/README.md

```diff
@@ -23,19 +23,32 @@ On the `developer.airbitz.co` page, scan the QR code using the Airbitz Mobile Ap
 Install from npm
 
     npm install airbitz-core-js-ui --save
-    
+
 Dependencies required in packages.json
 ```javascript
     "dependencies": {
-        "airbitz-core-js-ui": "git+ssh://git@github.com/Airbitz/airbitz-core-js-ui.git#react-native",
-        "airbitz-core-react-native": "git+ssh://git@github.com/airbitz/airbitz-core-react-native.git",
-        "react": "16.0.0-alpha.12",
-        "react-native": "0.47.0",
-        "rfc4648": "^1.0.0",
-        "secure-random": "^1.1.1"
+       "edge-login": "^0.5.6",
+        "material-ui": "^0.20.0",
+        "moment": "~2.14.1",
+        "native-base": "^2.3.9",
+        "react": "~15.4.2",
+        "react-dom": "^15.4.2",
+        "react-native": "^0.47.0",
+        "react-native-keyboard-aware-scroll-view": "^0.4.1",
+        "react-native-linear-gradient": "^2.0.0",
+        "react-native-mail": "^3.0.5",
+        "react-native-material-textfield": "git://github.com/Airbitz/react-native-material-textfield.git",
+        "react-native-modal": "^4.1.0",
+        "react-native-qrcode": "^0.2.6",
+        "react-native-vector-icons": "^4.4.2",
+        "react-redux": "~4.4.5",
+        "redux": "~3.5.2",
+        "redux-thunk": "~2.1.0",
+        "sprintf-js": "^1.0.3",
+        "zxcvbn": "^4.4.2"
```

Comment:
Do we need to bother the user with this list, since we are now a proper package with dependencies?